08-projects.Rmd

```diff
@@ -4046,9 +4048,9 @@ head airports.csv
 
 **Note:** You may be wondering *why* we are asking you to do this. This sort of situation (where you need to deal with quotes) happens a lot! It's important to practice and learn ways to fix these things.
 
-**Hint:** You could use `gsub` within `awk` to replace '"' with ''.
+**Hint:** You could use `gsub` within `awk` to replace '"' with ''. You can find how to use `gsub` [here](https://www.gnu.org/software/gawk/manual/html_node/String-Functions.html).
 
-**Hint:** If you leave out the column number argument to `gsub` it will apply the substitution to every field in every column.
+**Hint:** If you leave out the column number argument to `gsub` it will apply the substitution to every field in every column. 
```

Comment:
Ah, I did not mean to delete this space! Dear @kevinamstutz , is there a way that I could revert this change? I hope it would not lead to some strange paragraph organization.

Reply:
The only way I know would be to checkout your pull request branch and remove the extra space and then commit the change :) No need though, an extra space won't hurt a thing!